lizard_auth_client/views.py

```diff
@@ -275,6 +298,12 @@ def get_request_token_and_determine_response(domain=None):
         'key': settings.SSO_KEY,
         'domain': domain,
     }
+
+    # If this is true, the SSO server does not force a login and only logs
+    # in a user that is already logged in on the SSO server.
+    if attempt_login_only:
+        params['return_unauthenticated'] = True
```

Comment:
Dit zal met de lizard-auth-server ook te maken hebben. De parameter naam vind ik raar. Je wilt zeggen dat het OK is om ongeauthenticeerd terug te komen. Maar nu klinkt het alsof je dat vereist. "attempt_login_only" dekt de lading veel beter.